### ISSUE TITLE
JetBrain's Plugin does not work on Pycharm

### ISSUE THREAD

**Issue by ni-todo-spot** [metalbear-co/mirrord#828]
### Bug Description

Downloaded the plugin via the marketplace. When I try to debug a python script via the IDE's debugger (the mirrord button was already pressed), the procces starts but after couple of seconds I get an error.

### Steps to Reproduce

1. Hit the mirrord plugin button
2. Hit the debug button of Pycahrm
3. Wait for the debug session to start

### Your operating system and version

Darwin roberts-MacBook-Pro.local 22.1.0 Darwin Kernel Version 22.1.0: Sun Oct  9 20:14:54 PDT 2022; root:xnu-8792.41.9~2/RELEASE_X86_64 x86_64

### Local process

hitting the run / debug button in the ide

**Comment by ni-todo-spot** [comment]
additional info github prevented me from posting due to body length:

Used a py script for reproduction:
```python
import redshift_connector as redshift

PARAMETERS = {
    "numeric_to_float": True,
    "user": "user",
    "password": "pass",
    "host": "host",
    "port": 5439,
    "database": "db",
}

if __name__ == "__main__":
    sql = "SELECT 1 as num, true as bool_, 'a' as string_, 4.4 as float_"
    with redshift.connect(**PARAMETERS) as connection:
        connection.autocommit = True
        with connection.cursor() as cur:
            results = cur.execute(sql)
            print(results.fetchall())

```

requirements.txt contents:
```
asn1crypto==1.5.1
beautifulsoup4==4.11.1
boto3==1.26.25
botocore==1.29.25
certifi==2022.12.7
charset-normalizer==2.1.1
idna==3.4
jmespath==1.0.1
lxml==4.9.1
packaging==22.0
python-dateutil==2.8.2
pytz==2022.6
redshift-connector==2.0.909
requests==2.28.1
s3transfer==0.6.0
scramp==1.4.4
six==1.16.0
soupsieve==2.3.2.post1
urllib3==1.26.13
```


pycharm version:
```
PyCharm 2022.3 (Community Edition)
Build #PC-223.7571.203, built on November 30, 2022
Runtime version: 17.0.5+1-b653.14 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 13.0.1
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 8
Metal Rendering is ON
Registry:
    debugger.watches.in.variables=false

Non-Bundled Plugins:
    com.metalbear.mirrord (3.13.0)
    com.intellij.plugins.watcher (223.7571.113)
    Docker (223.7571.212)
    uk.co.hadoopathome.intellij.avro.intellijavroviewer (3.1.0)
    ru.adelf.idea.dotenv (2022.3)
    net.seesharpsoft.intellij.plugins.csv (3.0.1-223)
```

mirrord version: ```mirrord 3.13.0```

successful cli execution:
```
(venv) robert@roberts-MacBook-Pro mirrord_pycharm %  mirrord exec -l TRACE -a test -n test -t deployment/foo /Users/robert/Projects/mirrord_pycharm/venv/bin/python main.py
⠦ mirrord cli starting
  ✓ ready to launch process
    ✓ layer extracted
    ✓ agent pod created
    ✓ pod is ready
([1, True, 'a', 4.4],)
```

**Comment by aviramha** [comment]
I'm afraid the log file might contain (your) secrets - can you remove it?

**Comment by eyalb181** [comment]
Hi @ni-todo-spot, this happened with the default mirrord configuration in Pycharm, right? Does it also happen if you check the "TCP Steal Traffic" option?

**Comment by ni-todo-spot** [comment]
Hey @eyalb181 , 
Regarding your first question - I used the default configuration.
I've also tried the "TCP Steal Traffic" option but got the same result.

Thanks, Appreciate your time!

**Comment by eyalb181** [comment]
Thanks! So "TCP Outgoing Traffic" is checked, right? Does it happen both when it's checked and when it isn't?

**Comment by ni-todo-spot** [comment]
Yep - Ran with the following configuration and got same result (added the py error for the protocol):

```
struct.error: unpack_from requires a buffer of at least 5 bytes for unpacking 5 bytes at offset 0 (actual buffer size is 0)
```

![image](https://user-images.githubusercontent.com/89446609/207543699-1e33e967-85e6-45bb-9cd3-6045e68ce587.png)

**Comment by eyalb181** [comment]
But just to make sure - you get the same error when you uncheck TCP Outgoing Traffic and UDP Outgoing Traffic?
Either way - would you be open to have a debug session with us? We're having trouble reproducing the issue on our end. If it sounds good to you, you can schedule the session on our [calendly](https://calendly.com/eyal-metalbear/debug-session?month=2022-12).

**Comment by eyalb181** [comment]
@ni-todo-spot We had another question in the meanwhile - what port are you using to access Redshift?

**Comment by ni-todo-spot** [comment]
> @ni-todo-spot We had another question in the meanwhile - what port are you using to access Redshift?

5439

**Comment by aviramha** [comment]
Fixed in #780
